aiohttp/http_parser.py

```diff
@@ -669,6 +669,7 @@ def feed_data(
     ) -> Tuple[List[Tuple[RawResponseMessage, StreamReader]], bool, bytes]:
         if SEP is None:
             SEP = b"\r\n" if DEBUG else b"\n"
+            assert SEP is not None
```

Comment:
This shouldn't be added. It's a regression that has been fixed in the next mypy version. (I thought I closed this PR while waiting for the next release..).